Examples/STM32/uart_cli/README.md

```diff
@@ -54,6 +54,19 @@ Alternatively, use an IDE to compile and flash the project into the NUCLEO board
 2. Select the bus configuration (**SPI or SDIO**). Make sure it matches the switch position on the WF(M)200 expansion board.
 3. Build the project.
 4. Flash the project to the NUCLEO-F429ZI board.
+5. Please be aware that:
+-  if you are using newest versions of IAR Embedded Workbench, the output messages will be displayed on the **Terminal I/O window** of the IAR Embedded Workbench by default when you run the project through the **debug mode**. 
```

Comment:
"newest " biết đâu nó fix cái lỗi này thì sao em?
A đề xuất chú chèn cái link page thông báo "lỗi" phải tự implement printf() khi migrate từ IAR phiên bản cũ sang phiên bản mới. 
https://www.iar.com/knowledge/support/technical-notes/compiler/migrating-projects-with-the-custom-fputc-function/